Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -229,6 +229,10 @@ internal class CustomerIOImplementation: CustomerIOInstance {
             logger.info("no profile identified, so not registering device token to a profile")
             return
         }
+        guard !identifier.isEmpty else {
+            logger.error("not registering device token to a profile since profiles cannot be created without an identifier. Empty string was provided as the identifier, therefore, request to identify profile in the SDK is ignored. Try identifying a profile again with a non-empty identifier value.")
+            return
+        }
```

Comment:
Sorry I didn't notice this in my first PR review, but I think that we want the empty identifier check to go into the identify profile function. Not in the register device token function. 

I don't think we need to check if the `profileStore.identifier` is empty because I don't think we should set `profileStore.identifier` with a non-empty value in the first place (in the SDK identify profile function).

Reply:
I think `identifier.isEmpty` check should go [in the first line of the identify function](https://github.com/customerio/customerio-ios/blob/6e53c1b1688a913666869372168acdcc5663abfb/Sources/Tracking/CustomerIOImplementation.swift#L82).

Reply:
That makes sense. I have added a check to identify function and removed from registerDeviceToken method.